central/image/datastore/store/dackbox/store_impl.go

```diff
@@ -158,6 +160,11 @@ func (b *storeImpl) GetImageMetadata(_ context.Context, id string) (image *stora
 	return image, image != nil, err
 }
 
+func (b *storeImpl) GetManyImageMetadata(ctx context.Context, id []string) ([]*storage.Image, []int, error) {
+	utils.Must(errors.New("Unexpected call to GetManyImageMetadata when running on Postgres"))
```

Comment:
[nit] This error message confused me for a minute until I realized it's saying "unexpected call to GetManyImageMetadata in dackbox when running on Postgres". Maybe update the message to make it clear that it's about the fact that it's in dackbox that's the issue.